modules/caddyhttp/fileserver/browse.go

```diff
@@ -66,8 +66,15 @@ type Browse struct {
 	//   - `sort size` will sort by size in ascending order
 	// The first option must be `sort_by` and the second option must be `order` (if exists).
 	SortOptions []string `json:"sort,omitempty"`
+
+	// FileLimit limits the number of up to n DirEntry values in directory order.
```

Comment:
Should we document that -1 means no limit? (And 0 is default limit, which is currently 10,000)